src/main/java/net/greatstart/validators/InvestmentValidationService.java

```diff
@@ -15,23 +17,30 @@
             + "Max possible investment is: $";
     private static final String WRONG_STEP_VALUE = MESSAGE
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "WRONG_STEP_VALUE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)